### PR TITLE
Style according to pep8

### LINKED ISSUE #178: Style pynwb according to pep8
For standardized styling projects should follow [pep8](https://www.python.org/dev/peps/pep-0008/).

### PR DESCRIPTION
**Pull request by dorukozturk**
Fixes #178 #180
Applies flake8 styling to pynwb. 
@jcfr PTAL.

### CODE REVIEW

File: src/pynwb/io/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```
Comment (jcfr) [inline src/pynwb/io/__init__.py]: I suggest to exclude specific error instead. E,g `#  noqa: EXXX`

**Review by jcfr — DISMISSED** [review]

**Comment by oruebel** [comment]
@dorukozturk is there a way to customize the flake8 checking? E.g., it would be nice to relax the 80 characters per line limit to something more reasonable.

**Comment by jcfr** [comment]
> it would be nice to relax the 80 characters per line limit to something more reasonable.

Indeed, length has already been  increased to 100. I suggest to bump it to 120

See https://github.com/NeurodataWithoutBorders/pynwb/pull/181/files#diff-380c6a8ebbbce17d55d50ef17d3cf906

**Comment by ajtritt** [comment]
I don't use punchcards and I got rid of my CRT monitor, so I can do 120 characters.

**Comment by oruebel** [comment]
I agree, 120 characters is much more reasonable.

**Comment by dorukozturk** [comment]
This is still work in progress, I push time to time to see if I broke something. I set the limit to 120 characters.

File: src/pynwb/form/spec/namespace.py [hunk 2]
```diff
@@ -256,15 +264,17 @@ def __get_spec_path(cls, ns_path, spec_path):
...
+            {'name': 'resolve', 'type': bool,
+             'doc': 'whether or not to include objects from included/parent spec objects', 'default': True},
             returns='a dictionary describing the dependencies of loaded namespaces', rtype=dict)
     def load_namespaces(self, **kwargs):
         """Load the namespaces in the given file"""
         namespace_path, resolve = getargs('namespace_path', 'resolve', kwargs)
         # load namespace definition from file
         if not os.path.exists(namespace_path):
-            raise FileNotFoundError("namespace file '%s' not found" % namespace_path)
+            raise OSError("namespace file '%s' not found" % namespace_path)
```
Comment (dorukozturk) [inline src/pynwb/form/spec/namespace.py]: See https://github.com/nimiq/promptastic/issues/17. FileNotFoundError is only a Python 3 Exception.

**Review by jcfr — DISMISSED** [review]

**Pull request merged**